### PR TITLE
Remove BaseBlockAdmin getPersistentParameters

### PR DESCRIPTION
**Pull request by Hanmac**
configurePersistentParameters is already below, and it isn't in 3.x

### CODE REVIEW

File: src/Admin/BaseBlockAdmin.php [hunk 1]
```diff
@@ -179,17 +179,6 @@ public function setContainerBlockTypes(array $containerBlockTypes): void
         $this->containerBlockTypes = $containerBlockTypes;
     }
 
-    public function getPersistentParameters(): array
```
Comment (eerison) [inline src/Admin/BaseBlockAdmin.php:182]: - is there any reason to remove this?
- have you checked if it has used in the code, for example: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Controller/BlockAdminController.php#L86
- ~~If you want to remove this method in `4.x`, you should deprecate in `3.x` ?!~~ remove method without any deprecate message or annotation is fine too right @VincentLanglet ?

Comment (Hanmac) [reply src/Admin/BaseBlockAdmin.php:182]: * reason: the parent method is or is going to be final:
https://github.com/sonata-project/SonataAdminBundle/blob/29fc2f711f5eaf8108605f9d09b3c9f44ad374f3/src/Admin/AbstractAdmin.php#L1457
* the method exist in 4.x but for some reason not in 3.x
* the correct method  configurePersistentParameters already exist in 3.x

Comment (VincentLanglet) [reply src/Admin/BaseBlockAdmin.php:182]: Indeed there is
```
    protected function configurePersistentParameters(): array
    {
        if (!$this->hasRequest()) {
            return [];
        }
        return [
            'type' => $this->getRequest()->get('type'),
        ];
    }
```
doing the job

**Review by VincentLanglet — APPROVED** [review]

File: src/Admin/BaseBlockAdmin.php (hunk 1 above)
Comment (eerison) [reply src/Admin/BaseBlockAdmin.php:182]: well in theory, if you remove this method, it'll throw an exception, have you tested this?

Comment (VincentLanglet) [reply src/Admin/BaseBlockAdmin.php:182]: It's already implemented in the AbstractAdmin

**Pull request merged**